test/e2e/network_test.go

```diff
@@ -381,7 +391,9 @@ func (tc *testContext) createWindowsServerDeployment(name string, command []stri
 func (tc *testContext) waitUntilDeploymentScaled(name string) error {
 	var deployment *appsv1.Deployment
 	var err error
-	for i := 0; i < retryCount; i++ {
+	retryIntervalForScaling := 2 * time.Minute
```

Comment:
Could you add a comment saying why the retry interval/count has to be longer here?

Reply:
And why 2 minutes?

Reply:
Nit: I would not use variables here as they are single use and where they are being used makes it obvious enough without the naming of the variables.